templates/.github/workflows/go-test.yml

```diff
@@ -37,6 +39,9 @@ jobs:
         uses: protocol/multiple-go-modules@v1.0
         with:
           run: go test -v -race ./...
+      - name: Collect coverage files
+        shell: bash
+        run: echo "COVERAGES=`find . -type f -name 'module-coverage.txt' | tr -s '\n' ',' | sed 's/.$//'`" >> $GITHUB_ENV
```

Comment:
I think you need to change -coverprofile to use module-coverage.txt now?

Reply:
FYI, best to use `$(cmd)` rather than `` `cmd` ``, as the latter syntax has been deprecated for a long time by the POSIX shell spec.

Reply:
One last nit: if the sed is to remove the trailing comma, I think `,$` would be more obvious.

Reply:
> I think you need to change -coverprofile to use module-coverage.txt now?

I had this in my test repo, but of course I forgot to apply it here... I'm glad I have reviewers who pay close attention :)

Reply:
Fixed and applied your suggestions.